integration-tests/kafka-streams/src/test/java/io/quarkus/it/kafka/streams/KafkaStreamsPropertiesTest.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.it.kafka.streams;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.StreamsConfig;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTestResource(KafkaTestResource.class)
+@QuarkusTest
+public class KafkaStreamsPropertiesTest {
+
+    @Inject
+    KafkaStreams streams;
+
+    @Test
+    public void testProperties() throws Exception {
+        // reflection hack ... no other way to get raw props ...
+        Field configField = KafkaStreams.class.getDeclaredField("config");
+        configField.setAccessible(true);
+        StreamsConfig config = (StreamsConfig) configField.get(streams);
```

Comment:
How about adding a package visible accessor?

Reply:
What do you mean?

Reply:
Ah, seems you get a field from the upstream `KafkaStreams` object. I thought about having a getter in our own object (topology manager), where we could expose a package-visible getter for testing purposes.